### PR TITLE
Prepare for v1.0.0 release

### PR DESCRIPTION
**Pull request by KamilSztandur**
image

### CODE REVIEW

File: packages/bloc_presentation/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 1.0.0
+
+- **BREAKING**: Remove `useBlocPresentationListener` hook and replace it with `BlocPresentationListener` widget.
```
Comment (shilangyu) [inline packages/bloc_presentation/CHANGELOG.md:3]: It wasn't replaced, the widget was already there

**Review by shilangyu — APPROVED** [review]

File: packages/bloc_presentation/CHANGELOG.md (hunk 1 above)
Comment (Albert221) [reply packages/bloc_presentation/CHANGELOG.md]: Actually, replace it with `useOnStreamChange` from `flutter_hooks`. But this is changelog. We don't say to replace, we say it's been replaced. We can form a next sentence suggesting to replace

Comment (KamilSztandur) [reply packages/bloc_presentation/CHANGELOG.md:3]: Right. It was totally reimplemented so I assumed wrongly it's something new

File: packages/bloc_presentation/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 1.0.0
+
+- **BREAKING**: Removed `useBlocPresentationListener` hook and reimplemented `BlocPresentationListener` widget because of `flutter_hooks` dependencies removal.
```
Comment (Albert221) [inline packages/bloc_presentation/CHANGELOG.md:3]:
```suggestion
- **BREAKING**: Removed `useBlocPresentationListener` hook. Use `useOnStreamChanged` from `flutter_hooks` instead.
- Removed dependency on `flutter_hooks`.
```

File: packages/bloc_presentation/README.md [hunk 3]
```diff
@@ -100,6 +100,12 @@ useOnStreamChange(
 
 [Here it is.](/packages/bloc_presentation/example/lib)
 
+### Testing
+
+| Package                                                 | Documentation                                         | pub                                                                                                                | CI                                                                                 |
+|---------------------------------------------------------|:-----------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------:|:----------------------------------------------------------------------------------:|
+| [`bloc_presentation_test`][bloc_presentation_test-link] | [Documentation][bloc_presentation_test-documentation] | [![bloc_presentation_test pub.dev badge][bloc_presentation_test-pub-badge]][bloc_presentation_test-pub-badge-link] | [![][bloc_presentation_test-build-badge]][bloc_presentation_test-build-badge-link] |
```
Comment (Albert221) [inline packages/bloc_presentation/README.md:107]: These link aliases aren't defined...

image

But please mention the package in text. A simple sentence to direct consumers of this package would be enough.

File: packages/bloc_presentation/README.md [hunk 4]
```diff
@@ -100,6 +100,12 @@ useOnStreamChange(
 
 [Here it is.](/packages/bloc_presentation/example/lib)
 
+### Testing
```
Comment (Albert221) [inline packages/bloc_presentation/README.md:103]: Let's make this 2nd level heading

File: packages/bloc_presentation/README.md [hunk 5]
```diff
@@ -100,6 +100,12 @@ useOnStreamChange(
```
Comment (Albert221) [inline packages/bloc_presentation/README.md:100]: The example above belongs rather to the "Usage" section. Please move it under it

**Pull request merged**